python/ray/runtime_context.py

```diff
@@ -11,28 +11,107 @@ def __init__(self, worker):
         assert worker is not None
         self.worker = worker
 
+    def get(self):
+        """Get a dictionary of the current_context.
+
+        The value contains None if the field is not available.
+        For example, if you invoke this method from a driver,
+        task_id or actor_id is not available. In this case,
+        the dictionary will contain {"actor_id": None, "task_id": None}
```

Comment:
Shouldn't this just not contain those keys? That would probably work better for using w/ the metrics API, right?

Reply:
Hmm that's a good point. You are saying we can do sth like

```
metric.record(1, tags=runtime_context().get())
```

right? I think it makes sense. Let me fix it.